dlc-manager/src/channel/signed_channel.rs

```diff
@@ -297,6 +297,17 @@ typed_enum!(
             /// Whether the local party initiated the closing of the channel.
             is_initiator: bool,
         },
+        /// A [`SignedChannel`] is in `SettledClosing` state when a settle transaction has been
+        /// broadcast, and we are waiting to finalize the closing of the channel by claiming one of
+        /// its outputs.
+        SettledClosing {
+            /// The settle transaction that was broadcast.
+            settle_transaction: Transaction,
+            /// Indicates whether the local party originally offered to settle the channel or not.
+            is_offer: bool,
```

Comment:
❓ whats the difference between `is_offer` and `is_initiator` ?

Reply:
I think the docs explain this: `is_offer` indicates if the local party initiated the _settle_ protocol; `is_initiator` indicates if the local party triggered the force-close.

Reply:
Thanks